src/main/java/javax/validation/ClockProvider.java

```diff
@@ -0,0 +1,42 @@
+/*
+* Copyright 2009-2012, Red Hat, Inc. and/or its affiliates, and individual contributors
```

Comment:
2016. In parallel need to check with @emmanuelbernard whether we can use the simplified form as in the RI, which avoids this issue to begin with.